unsteady/test_cases/trench_1d_der/trench_dual_exp_general.py

```diff
@@ -0,0 +1,181 @@
+from firedrake_adjoint import *
```

Comment:
Provide a docstring

Reply:
file included by accident and so now removed